qiskit/providers/basicaer/qasm_simulator.py

```diff
@@ -141,12 +145,19 @@ def __init__(self, configuration=None, provider=None):
         self._number_of_qubits = 0
         self._shots = 0
         self._memory = False
-        self._initial_statevector = self.DEFAULT_OPTIONS["initial_statevector"]
-        self._chop_threshold = self.DEFAULT_OPTIONS["chop_threshold"]
+        self._initial_statevector = self.options.get('initial_statevector')
+        self._chop_threshold = self.options.get("chop_threashold")
         self._qobj_config = None
         # TEMP
         self._sample_measure = False
 
+    @classmethod
+    def _default_options(cls):
+        return Options(shots=1024, memory=False,
+                       initial_statevector=None, chop_threshold=1e-15,
+                       allow_sample_measuring=False, seed_simulator=None,
```

Comment:
I think `allow_sample_measuring` should be True by default otherwise it will be very slow (1000x slower for a 1000-shot simulation).

Reply:
I was just mirroring what was the default before. I'll change it here and add an upgrade release note about the change

```suggestion
                       allow_sample_measuring=True, seed_simulator=None,
```

Reply:
I actually think we'll be ok without a release note for this. Looking at the code for checking the option: https://github.com/Qiskit/qiskit-terra/blob/master/qiskit/providers/basicaer/qasm_simulator.py#L340-L378 it would default to ``True`` if nothing was set if the circuit is capable of using sample measuring. So I think putting either value here doesn't matter too much as it doesn't really change the behavior.